esp32-wroom-rp/src/protocol/operation.rs

```diff
@@ -1,6 +1,7 @@
+use heapless::Vec;
+
 use crate::protocol::{NinaAbstractParam, NinaCommand};
```

Comment:
`use super::protocol::...` ?

Reply:
Updated 1a44b763440c00eb7f0e9886d57f217674e0500d

Reply:
Nice catch @dilyn-corner, I missed this one.